sdk/storage/storage-blob/test/node/sas.spec.ts

```diff
@@ -461,7 +461,7 @@ describe("Shared Access Signature (SAS) generation Node.js only", () => {
       {
         accessPolicy: {
           expiresOn: tmr,
-          permissions: ContainerSASPermissions.parse("racwdl").toString(),
+          // permissions: ContainerSASPermissions.parse("racwdl").toString(),
```

Comment:
Is this by design?

Reply:
I am thinking about whether to delete the line or make it commented out. A comment can be referenced as a sample when we want to know how to provide the parametr later on.

Reply:
Minor: I see, Please add comments inline for the purpose.

---
In reply to: [380053526](https://github.com/Azure/azure-sdk-for-js/pull/7414#discussion_r380053526) [](ancestors = 380053526)

Reply:
Then I'd rather delete this line.